### ISSUE TITLE
Implement calendar component in quo2 preview

### ISSUE THREAD

**Issue by cammellos** [status-im/status-react#16377]
Create the quo2 component calendar as per designs:

https://www.figma.com/file/HKncH4wnDwZDAhc4AryK8Y/Wallet-UI-for-Mobile?node-id=780%3A214042&mode=dev

Should be implemented according to best practices
https://github.com/status-im/status-mobile/blob/develop/doc/new-guidelines.md

Include component spec tests
https://github.com/status-im/status-mobile/blob/develop/doc/component-tests-overview.md

Communicate with design team before starting to make sure there is no unknown requirements and to understand the interactions
In scope:

1) Day input
2) Calendar itself

Out of scope:
1) Hour input is out of scope (anything below and including "Select hour" is out of scope).

**Comment by super5508** [comment]
👋
